browser/ui/views/brave_actions/brave_shields_action_view.cc

```diff
@@ -116,10 +119,16 @@ BraveShieldsActionView::GetImageSource() {
       badge = std::make_unique<IconWithBadgeImageSource::Badge>(
           badge_text, SK_ColorWHITE, kBadgeBg);
     }
+
+    if (is_enabled) {
+      SetTooltipText(l10n_util::GetStringFUTF16(IDS_BRAVE_SHIELDS_ICON_TOOLTIP,
+        base::ASCIIToUTF16(std::to_string(count))));
```

Comment:
i wonder if there's a helper to convert `int` to `u16` directly.

Reply:
Maybe `std::u16string GetStringFUTF16Int(int message_id, int a);` is for you?

Reply:
nice, works